internal/controllers/router/tests/import/02-create-router.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: openstack.k-orc.cloud/v1alpha1
+kind: Router
+metadata:
+  name: import-external
+spec:
+  cloudCredentialsRef:
+    cloudName: openstack
+    secretName: openstack-clouds
+  managementPolicy: managed
+  resource:
+    name: import-external
```

Comment:
How do we manage to import this resource, as it doesn't have the `tag1` tag the import filter is requesting? This would be a bug.

Reply:
I was not able to reproduce.